### DISCUSSION TITLE
Using parameters in Studio causes a syntax warning

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#13493] [Q&A]
When I use RQL parameters in Studio, a warning icon is shown. It doesn't happen after using the "Run Query" shortcut from Traffic Watch, but after any modification, the warning appears.

![image](https://user-images.githubusercontent.com/7997055/149770170-a8228377-5004-4aac-9110-0e67fa29dae5.png)

This happens on 5.3.2 and 5.3.100-nightly-20220107-1334 (currently deployed on live-test).

**Comment by ayende** [comment]
@ml054

**Reply by arekpalinski** [reply]
@krivulcik Thanks for the report. @ml054 is currently away but he'll check it once he's back for sure.

**Reply by krivulcik** [reply]
Thanks for the information. The issue doesn't prevent any functionality from working. It only displays an incorrect warning, so it's nothing critical, only cosmetic, I would say.

**Reply by arekpalinski** [reply]
Yeah. Just wanted to let you know :)

**Reply by ml054** [reply]
@krivulcik Thanks for report. Created issue: https://issues.hibernatingrhinos.com/issue/RavenDB-17958
